lib/nimbusRecipeCollection.ts

```diff
@@ -1,12 +1,37 @@
 import { types } from "@mozilla/nimbus-shared"
 import { NimbusRecipe } from "../lib/nimbusRecipe"
+import { BranchInfo, RecipeInfo, RecipeOrBranchInfo } from "@/app/columns"
+import { getCTRPercent } from "./looker"
+
 type NimbusExperiment = types.experiments.NimbusExperiment
 
 type NimbusRecipeCollectionType = {
   recipes: Array<NimbusRecipe>
   fetchRecipes: () => Promise<Array<NimbusRecipe>>
 }
 
+/**
+ * @returns an array of BranchInfo with updated CTR percents for the recipe
+ */
+async function updateBranchesCTR(recipe: NimbusRecipe): Promise<BranchInfo[]> {
+  return await Promise.all(
+    recipe.getBranchInfos().map(
+      async (branch: BranchInfo): Promise<BranchInfo> => {
```

Comment:
Please rename this to branchInfo, since there's already a thing called branches inside nimbus recipes that is different.